app/redux/wallet/actions.js

```diff
@@ -166,6 +214,8 @@ export const saveNewWallet = ({ mnemonic, salt = cryptoConsts.DEFAULT_SALT }: {
     dispatch(setMnemonic({ mnemonic: resolvedMnemonic }));
     dispatch(setCurrentAccount({ index: 0 }));
     dispatch(setTransactions({ transactions }));
+    dispatch(setContacts({ contacts }));
+    dispatch(setLastUsedAddresses({ lastUsedAddresses: lastUsedAddressesStub })); // TODO: to be replaced with actual data
```

Comment:
explanation in TODO is wrong - there are no latest used addresses in freshly created wallet

Reply:
ok.